packages/devtools-client-adapters/src/chrome/create.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import type { Location } from "../types";
+import type { ServerLocation } from "./types";
+
+function fromServerLocation(serverLocation: ServerLocation): ?Location {
+  if (serverLocation) {
```

Comment:
I'm guessing you needed the guard here but it feels like the `serverLocation` isn't optional given how the params are done.